### ISSUE TITLE
Weird formatting with `:hiccup` + `wrap?`

### ISSUE THREAD

**Issue by RolT** [kkinnear/zprint#204]
Hello,

I have a weird behaviour while using the style `:hiccup` and `{wrap? true}` for vectors.

With the source file:
```clojure
[a {:a (let [val1 "eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee" val2 "eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"]
         (str val1 val2))}]
```

And this zprint config:
```edn
{:style [:hiccup]
 :width 100
 :vector {:wrap? true}}
 ```

I'm getting this result:

```clojure
[a {:a (let [val1 "eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee" val2
             "eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"]
         (str val1 val2))}]
```

I would expect this formatting:
```clojure
[a {:a (let [val1 "eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"
             val2 "eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"]
         (str val1 val2))}]
```

Any idea on how to solve this ?

**Comment by kkinnear** [comment]
Sorry it is taking me so long on this one, I was busy with another issue.  I can certainly reproduce this, and I would expect what you showed above.  It is far from clear what is going on, so I'll dig deeper into it and get back to you soon.

**Comment by kkinnear** [comment]
Thanks for noticing and reporting this issue!  It was a remarkably subtle problem that I have now found and fixed, and the fix will be available in the upcoming `1.2.0` release.  I spent some time looking for a workaround but was unable to find one, so until `1.2.0` is available you will either have to live with this, or skip using `:style :hiccup` until `1.2.0` is available.

Thanks for boiling this down to a small example, it saved me a lot of time.  There are now several new tests for this problem, so we don't backslide on this fix.  Thanks again!
